director/director.go

```diff
@@ -758,6 +758,16 @@ func registerServeAd(engineCtx context.Context, ctx *gin.Context, sType server_s
 	ctx.Set("serverName", adV2.Name)
 	ctx.Set("serverWebUrl", adV2.WebURL)
 
+	// Iterate over each advertised namespace and join the paths together
+	// into a string where each path is separated by a space
+	// i.e. "<path> <path> <path>"
+	var namespacePaths string
+	for _, namespace := range adV2.Namespaces {
+		path := namespace.Path
+		namespacePaths = fmt.Sprintf("%s %s", namespacePaths, path)
+	}
+	ctx.Set("namespacePaths", namespacePaths)
```

Comment:
You set the context to be `namespacePaths` but later you read `namespacePrefixes` instead. I'd suggest we use `namespacePrefixes` for both